components/fieldgroups/provider/healthcare-provider-details.schema.json

```diff
@@ -0,0 +1,163 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/healthcare-provider",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Healthcare Provider",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["healthcare_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/provider"],
+  "description": "Details of an individual health professional or a health facility organization licensed to provide health care diagnosis and treatment services including medication, surgery and medical devices.",
```

Comment:
Details of an individual health professional or a health facility organization licensed to provide health care diagnosis and treatment services.